docs/en/advanced-concepts/wda-custom-server.md

```diff
@@ -55,15 +55,13 @@ so you'll see the icon of WebDriverAgentRunner application on the springboard.
 
 WebDriverAgent application acts as a REST server, which proxies external API requests to native XCTest calls
 for your application under test. The server address will be _localhost_ if you run your tests on Simulator
-or the actual phone IP address in case of real device.
-We use [appium-ios-device](https://github.com/appium/appium-ios-device) to route network requests
+or the actual phone IP address in case of real device. Appium uses [appium-ios-device](https://github.com/appium/appium-ios-device) to route network requests
 to a real device from _localhost_ via USB, which means one can use this tool to unify WDA network
 address for Simulator and for real device.
 
 You can use [appium-ios-device](https://github.com/appium/appium-ios-device) to connect to
 a remote device requiring the module from your JavaScript code as same as Appium.
-Alternatively, you can also use _iproxy_ to handle WebDriverAgent outside Appium.
-It is available via `node install -g iproxy`.
+Alternatively, you can also use [iproxy](https://github.com/libimobiledevice/libusbmuxd#iproxy), [go-ios](https://github.com/danielpaulus/go-ios) or [tidevice](https://github.com/alibaba/taobao-iphone-device) to handle WebDriverAgent process outside Appium. For instance, `iproxy` launches a WebDriverAgent process via `node install -g iproxy`.
```

Comment:
outside of Appium
For instance, `iproxy` launches a WebDriverAgent process via `node install -g iproxy` -> For instance, `iproxy` could be installed using npm: `npm install -g iproxy`

Reply:
oops...